docs/conf/vilnius/2019/tickets.rst

```diff
@@ -0,0 +1,98 @@
+:template: {{year}}/generic.html
+
+
+Tickets
+=======
+
+{% if flagticketsonsale %}
+
+**Tickets are on sale now!**
+
+{% else %}
+
+**Tickets are sold out!**
+
+{% endif %}
+
+Each ticket includes:
+
+* Entry to all conference events and activities
+* Breakfast, snacks, and lunch on all conference days
+* Welcome Reception and Social Event with light snacks and free drinks
+* Wifi throughout the event
+* Meeting lots of fantastic people in a spacious, inviting venue
+
+{% if flaghasshirts %}
+
+.. class:: ticket
+
+**Official Conference Shirts**
```

Comment:
remove this (no shirts or any other swag for first-year unless we get surprised with a ton of sponsorship)

Reply:
Presumably we'll do print on demand though right?